source/fundamentals/authentication/mechanisms.txt

```diff
@@ -233,28 +233,31 @@ The driver checks for your credentials in the following sources in order:
 
 The ``X.509`` authentication mechanism uses
 :wikipedia:`TLS <Transport_Layer_Security>` with X.509 certificates to
-authenticate your user, identified by the distinguished name (DN) of your
-client certificate. For more information on determining the *subject* name from
-the X.509 certificate, see the MongoDB manual :manual:`X.509 tutorial
-</tutorial/configure-x509-client-authentication/#add-x-509-certificate-subject-as-a-user>`.
+authenticate by retrieving the distinguished name (DN) from the
+client certificate.
 
 You can specify this authentication mechanism by setting the following
-parameters of your
-:manual:`connection string </reference/connection-string/>`:
+parameters of your :manual:`connection string </reference/connection-string/>`:
 
-- Set the ``authMechanism`` parameter to the value ``MONGODB-X509``
-- Set the ``tls`` parameter to the value ``true``
+- Set the ``authMechanism`` parameter to ``MONGODB-X509``
+- Set the ``tls`` parameter to ``true``
 
 Pass the location of your client certificate file as the value of
 ``tlsCertificateKeyFile`` as a parameter of the connection URI.
 
 .. important::
-   Always **URI encode** the username and certificate file path using the
-   ``encodeURIComponent`` method to ensure they are correctly parsed.
+   Always **URI encode** the certificate file path using the
+   ``encodeURIComponent`` method to ensure it is parsed correctly.
 
 .. literalinclude:: /code-snippets/authentication/x509.js
    :language: javascript
 
+.. note::
```

Comment:
Suggestion:

Based on the comments on the ticket, it's not needed for all the versions of the Node.js driver docs we maintain. I would omit this since it might cause users to ask why they might include it.

Reply:
Good call. I wasn't sure because the ticket says we might want to tell users that it is optional but requires URI encoding... But if there's really no use case where users need to supply the user, best to not reference it at all